wee_slack.py

```diff
@@ -4000,6 +4001,21 @@ def command_thread(data, current_buffer, args):
 
 command_thread.completion = '%(threads)'
 
+
+@slack_buffer_required
+@utf8_decode
+def command_broadcast(data, current_buffer, args):
+    """
+    /broadcast <text>
```

Comment:
You should write `/slack broadcast <text>` here, as it won't work as just `/broadcast`.